include/ast/module.h

```diff
@@ -161,5 +138,55 @@ class Module {
   /// @}
 };
 
+class CoreModuleSection : public Section {
+public:
+  /// Getter of content.
+  Span<const Module> getContent() const noexcept { return Content; }
+  std::vector<Module> &getContent() noexcept { return Content; }
+
+private:
+  std::vector<Module> Content;
+};
```

Comment:
Here, unlike nested component, we no need to wrap module again (no recursive definition)